pages/experiences/filecoin-wallet-balances.js

```diff
@@ -8,13 +8,20 @@ const EXAMPLE_CODE = `import * as React from 'react';
 import { FilecoinBalancesList } from 'slate-react-system';
 import { createPow } from "@textile/powergate-client";
 
-const PowerGate = createPow({ host: 'http://0.0.0.0:6002' });
-const { info } = await PowerGate.ffs.info();
+const PowerGate = createPow({ host: "http://pow.slate.textile.io:6002" });
 
 class Example extends React.Component {
+  componentDidMount = async () => {
+    const FFS = await PowerGate.ffs.create();
+    const token = FFS.token ? FFS.token : null;
+    PowerGate.setToken(token);
+    const { info } = await PowerGate.ffs.info();
+    this.info = info;
```

Comment:
* **Please fix**
* **Non blocking**

This is a bad habit. We want to keep the value updates within `this.setState` to pair better with the `React lifecycle`. Especially because `info` values map directly with rendered components, where with other values its sometime acceptable because updating them does not mean the component should `re-render` (if you were wondering the difference)